### DISCUSSION TITLE
Dropping Quiz Grades

### DISCUSSION THREAD

**Discussion by AjeethUConn** [cooperrc/advanced-dynamics#44] [Q&A]
Hello!

Prof. Cooper said in the most recent lecture something about dropping quiz grades. I looked through the syllabus and had difficulty finding information about that policy. Does anyone know where I can find information about that?

**Comment by dan21014** [comment]
I did not see this in the lecture, but I also couldn't find anything in the syllabus. Is this a case by case basis, or a general thing for all students?
